### PR TITLE
Set up a "Hello world" project with two apps

### PR DESCRIPTION
**Pull request by zewdu444**
### Set up a "Hello world" project with two apps (Backend)
 
- [x] setup  rails with API default.
- [x] Add Postgres database with greeting models.
- [x]  Add end point for rails API.
         [ http://127.0.0.1:3000/api/v1/greetings]( http://127.0.0.1:3000/api/v1/greetings)
- [x] On readme file React link has been placed [link](https://github.com/zewdu444/hello-react-front-end)
- [x]  Linter error has been fixed. 
 🔗 ----[React link](https://github.com/zewdu444/hello-react-front-end/pull/1)---- 🔗

### CODE REVIEW

**Comment by Kamba56** [comment]
Hi @zewdu444 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly add link to the frontend repo in the backend readme and link to the backend repo in the fronedn reademe.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: config/environment.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Load the Rails application.
+require_relative "application"
+require_relative "env"
+
+# Initialize the Rails application.
+ENV['DATABASE_PASSWORD'] = DATABASE_PASSWORD
+ENV['DATABASE_USERNAME'] = DATABASE_USERNAME
```
Comment (mckent05) [inline config/environment.rb]: **OPTIONAL** - These lines cause issues when I try setting up the database, I suggest removing them from your app, please fix this 🙏

**Review by mckent05 — APPROVED** [review]
Hi @zewdu444, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good jb, your backend renders a random greeting to the frontend 👏 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/environment.rb (hunk 1 above)
Comment (zewdu444) [reply config/environment.rb]: @mckent05 thanks for the feedback I fix the issue.

**Pull request merged**